.github/workflows/firebaseai.yml

```diff
@@ -0,0 +1,164 @@
+name: firebaseai
+
+on:
+  pull_request:
+    paths:
+    - 'FirebaseAI**'
+    - '.github/workflows/firebaseai.yml'
+    - 'Gemfile*'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read  # Needed for actions/checkout
+  actions: write # Needed for actions/cache (save and restore)
+
+jobs:
+  spm-package-resolved:
```

Comment:
Update based on recent spm ci refactor

Reply:
Looks like the `vertexai.yml` change isn't in yet: [`373052a` (#14739)](https://github.com/firebase/firebase-ios-sdk/pull/14739/commits/373052acc3ebeb20f4c8b016a2427b30f6874ec9)